wemake_python_styleguide/visitors/tokenize/comments.py

```diff
@@ -37,6 +37,10 @@
 )
 from wemake_python_styleguide.visitors.base import BaseTokenVisitor
 
+EMPTY_TOKEN = tokenize.TokenInfo(
```

Comment:
let's use `Final` from `typing` to indicate that this variable shouldn't be reassigned:

```suggestion
EMPTY_TOKEN: Final = tokenize.TokenInfo(
```

Reply:
What do you say for the `SENTINEL_TOKEN` name? This constant is mostly used as a sentinel in the end

Reply:
Looks like ``Final`` from ``typing`` is a ``3.8`` feature. How should I proceed?

Reply:
Ahh, yes, so let's use `typing_extensions` for compatibility with older python versions:

```python
from typing_extensions import Final
```